module-2/lab-df-calculation-and-transformation/your-code/challenge-1.ipynb

```diff
@@ -1589,6 +1589,3227 @@
    "outputs": [],
```

Comment:
-Print distinct values: 
```Python
distinct_val=pokemon.groupby(['Type 1','Type 2']).size().reset_index().rename(columns={0:'distinct count'})
print('Distinct values: ',len(distinct_val))
display(distinct_val.head(10))
```
- Cleanup Name that contain "Mega": Good one!!
- Another way of getting the guessed total column. When you have a huge number of columns in a real data set, It's not worthy to write down each column one by one. 
```Python
pokemon['Guessed Total'] = pokemon.loc[:,'HP':'Speed'].sum(axis = 1)
```